tests/foreman/api/test_role.py

```diff
@@ -245,65 +248,751 @@ def test_negative_update_role_taxonomies(self):
     @stubbed
     @tier1
     def test_positive_disable_filter_override(self):
-        """Unset override resets filter taxonomies
+        """Unsetting override flag resets filter taxonomies
 
         :id: eaa7b921-7c12-45c5-989b-d82aa2b6e3a6
 
         :steps:
 
-            1. Create role with overridden filter having different taxonomies
-               than its role.
-            2. Unset the override flag in above role filter
+            1. Create role with organization A and Location A
+            2. Create an overridden filter in role with organization B
+                and Location B
+            3. Set above filter override flag to False in role
+            4. Get above role filters
 
-        :expectedresults: The taxonomies of filters resets/synced to role
-            taxonomies
+        :expectedresults: On unsetting filter override, role filter should
+            return override flag as False
 
         :caseautomation: notautomated
 
         :CaseImportance: Critical
         """
 
     @stubbed
+    @tier1
+    def test_positive_create_org_admin_from_clone(self):
+        """Create Org Admin role which has access to most of the resources
+        within organization
+
+        :id: bf33b70a-25a9-4eb1-982f-03448d008ec8
+
+        :steps:
+
+            1. Create Org Admin role by cloning 'Organization admin' role which
```

Comment:
as I mentioned in my previous reviews to different PRs, it is not logical to have 1 step which is clone of test case description. I am suggesting to remove `steps` section at all for such situations